src/Tempest/Database/src/QueryBuilder.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Database;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_CLASS)]
+final readonly class QueryBuilder
```

Comment:
I'm not quite sure if the attribute is beneficial here. It could be a method in the `IsDatabaseModel` trait that classes could override.

Reply:
But I'm undecided on it. @aidan-casey @innocenzi, any feedback?

Reply:
I'm kind of indifferent on it. Attributes feel a little more Tempest-y, but that's pretty much all I've got. 😛

Reply:
True. Let's see what Enzo has to say. I'm fine with attributes if that what we all feel best about

Reply:
I agree with Aidan here. I like attributes better than overriding methods

Reply:
Ok, attributes it is

Reply:
@brendt @aidan-casey @innocenzi thank for your feedback. I'll keep it